### DISCUSSION TITLE
Show all withdrawable events and events where current xtz are locked [Suggestion]

### DISCUSSION THREAD

**Discussion by ztepler** [juster-finance/juster-app#18] [Ideas]
На текущий момент - единственный способ понять в каких ивентах у меня залочены ставки, это открыть https://app.juster.fi/profile и там будет большой список из разных ивентов в которых я участвовал. Сейчас очень сложно понять в каких именно ивентах залочены ставки.

Было бы хоршо иметь какое-то место (наверное как раз где-то в профиле), в котором можно было бы увидеть список из всех ивентов, в которых в данный момент залочена пользовательская (моя) ликвидность (без разницы ставка или provided liquidity).

Я представляю себе это в табличном виде в виде отдельных строк с зелёной кнопкой Withdraw [X] xtz для тех позиций, по которым можно сделать вывод и серой неактивной кнопкой Locked [X] xtz если вывод сделать пока нельзя. Те ивенты из которых вывод уже сделан / проигранные ивенты, по которым нельзя ничего извлечь - отображаться не должны. То есть эта должна быть таблица с только теми ивентами, в которых есть лок ликвидности.

Также можно было бы под этой таблицей сделать что-то типа [Withdraw all] кнопки, которая делает батч транзу с виздровом всех возможных ивентов.

Если требуется - я могу это нарисовать.
Это тоже предложение с невысоким приоритетом, которое я предлагаю его обсудить и реализовать в будущем, когда будет время, если вдруг никто не найдёт хорошей причины не делать это.

**Comment by xdeq** [comment]
- `WithdrawAll` is now available on the page for withdrawals
- Profile in redesign process
